NDTensors/ext/NDTensorsCUDAExt/indexing.jl

```diff
@@ -1,9 +1,15 @@
+using CUDA: CuArray
+using GPUArraysCore: @allowscalar
+using NDTensors: NDTensors
+using NDTensors.Expose: Exposed, expose, parent, unexpose
```

Comment:
Does `NDTensors.Expose` have it's own version of `parent`?

Reply:
Ideally we would use `Base.parent`, or if we needed a special version of `parent` I would think it would be defined in `NDTensors.TypeParameterAccessors`.

Reply:
No this is a mistake, there is just a version of `Base.parent(::Exposed)` in `Expose` it does not have its own version of the function. I removed it from the using list, thanks!